pkg/api/volume.go

```diff
@@ -117,7 +117,8 @@ func (v *VolumePortal) CreateVolume() {
 		Context:           ctx.ToJson(),
 	}
 	if _, err = v.CtrClient.CreateVolume(context.Background(), opt); err != nil {
-		log.Error("create volume failed in controller service:", err)
+		CreateVolumeError(ctx, &volume)
```

Comment:
Would it be better to rename this method to 'SetVolumeStatusErrored' or something such...since essentially, it is updating the status of the volume to errorred ?